bin/cml/runner.js

```diff
@@ -432,10 +432,12 @@ exports.builder = (yargs) =>
           'One or more user-defined labels for this runner (delimited with commas)'
       },
       idleTimeout: {
-        type: 'number',
-        default: 5 * 60,
+        type: 'string',
+        default: '5 minutes',
+        coerce: (val) =>
+          /^-?\d+$/.test(val) ? parseInt(val) : timestring(val),
```

Comment:
### Follow–up of https://github.com/iterative/cml/pull/882#discussion_r789878083
```suggestion
          val === 'never' ? -1 : (/^-?\d+$/.test(val) ? parseInt(val) : timestring(val)),
```

Reply:
timestring is by default seconds so this should work? (not tested)

```js
val === 'never' ? -1 : timestring(val)
```

Reply:
See https://github.com/mike182uk/timestring/issues/49 linked on https://github.com/iterative/cml/pull/882#issue-1110734881

Reply:
whaaat! treats them as seconds and the normal numbers treats them as milliseconds? 😅